WordPress/gutenberg
# Update "Versions in WordPress" page

## What?
This PR will updates the "[Versions in WordPress](https://developer.wordpress.org/block-editor/contributors/versions-in-wordpress/)" page.

## How?

I have referred to [the release list](https://wordpress.org/download/releases/) and added minor versions for all major versions, including past versions.

## Testing Instructions

Check out the new page.

https://github.com/WordPress/gutenberg/blob/update/version-in-wordpress/docs/contributors/versions-in-wordpress.md

P.S. I'd love to be able to automate this in some way in the future.
